diff --git a/wangle/CMakeLists.txt b/wangle/CMakeLists.txt
index 85ae066..9dfe5a0 100644
--- a/wangle/CMakeLists.txt
+++ b/wangle/CMakeLists.txt
@@ -74,7 +74,7 @@ if (gflags_FOUND)
 else()
   find_package(Gflags REQUIRED)
 endif()
-find_package(LibEvent MODULE REQUIRED)
+find_package(Libevent REQUIRED)
 find_package(DoubleConversion REQUIRED)
 find_package(Threads REQUIRED)
 if (UNIX AND NOT APPLE)
